store/storetest/link_metadata_store.go

```diff
@@ -196,10 +196,10 @@ func testLinkMetadataStoreTypes(t *testing.T, ss store.Store) {
 		require.IsType(t, &model.PostImage{}, received.Data)
 		assert.Equal(t, *(metadata.Data.(*model.PostImage)), *(received.Data.(*model.PostImage)))
 
-		result = <-ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
-		require.Nil(t, result.Err)
+		linkMetadata, err := ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
```

Comment:
probably you can use here the `received, err := ...` and remove the received assignation some lines later.